gwvolman/tasks.py

```diff
@@ -329,6 +330,12 @@ def remove_volume(self, instanceId):
         logging.info("Unmounting %s", dest)
         subprocess.call("umount %s" % dest, shell=True)
 
+    logging.info("Unmounting licenses")
+    if os.path.exists(MATLAB_LICENSE_PATH):
```

Comment:
I think you're missing volume path: `os.path.join(containerInfo['mountPoint'], MATLAB_LICENSE_PATH)` ?